google/cloud/aiplatform/jobs.py

```diff
@@ -2485,7 +2485,7 @@ def update(
             current_job.model_deployment_monitoring_objective_configs = (
                 ModelDeploymentMonitoringJob._parse_configs(
                     objective_configs,
-                    current_job.endpoint,
+                    aiplatform.Endpoint(current_job.endpoint),
```

Comment:
Please add a unit test to test this path.

Reply:
unit test added